### PR TITLE
re_trace

### PR DESCRIPTION
**Pull request by sreimers**

### CODE REVIEW

File: src/tmr/tmr.c [hunk 1]
```diff
@@ -112,6 +114,43 @@ void tmr_poll(struct list *tmrl)
...
+		DEBUG_WARNING("jiffies: clock_gettime() failed (%m)\n", errno);
+		return 0;
+	}
+
+	jfs  = (long)now.tv_sec * (uint64_t)1000000;
+	jfs += now.tv_nsec/1000;
+#endif
+
+	return jfs;
+}
```
Comment (alfredh) [inline src/tmr/tmr.c:151]: this is a useful function, is it possible to make it a separate PR ?

here is a similar function:

https://github.com/baresip/baresip/blob/master/src/timer.c#L29

some people also wanted to have clock_gettime.

after this patch have been merged, we can delete baresip/src/timer.c

Comment (sreimers) [reply src/tmr/tmr.c:151]: Done: https://github.com/baresip/baresip/pull/1141

I will update this PR.
